LayoutTests/ChangeLog

```diff
@@ -1,3 +1,14 @@
+2022-05-09  Kate Cheney  <katherine_cheney@apple.com>
+
+        Align ITP client side cookie cap with other script-writable storage
+        https://bugs.webkit.org/show_bug.cgi?id=240248
+        rdar://90468671
+
+        Reviewed by NOBODY (OOPS!).
+
+        * http/tests/resourceLoadStatistics/website-data-removal-for-site-without-user-interaction-expected.txt:
+        * http/tests/resourceLoadStatistics/website-data-removal-for-site-without-user-interaction.html:
```

Comment:
Wasn't there a test to check the expiry of client-side cookies for the previous cap functionality?

Reply:
Yes, I think I missed some tests that need to be updated. Looks like capped-lifetime-for-cookie-set-in-js.html, capped-lifetime-for-cookie-set-in-js-with-link-decoration-same-site.html, etc are some.

Reply:
Please keep them, but change them to make sure we don't cap after your change.